package-system/zlib/PackageInfo.json

```diff
@@ -1,5 +1,5 @@
 {
-    "PackageName" : "zlib-1.2.11-rev2-multiplatform",
+    "PackageName" : "zlib-1.2.11-rev3-multiplatform",
```

Comment:
Should the package name be updated to rev5 to reflect the package_version?

Reply:
This should be fixed now, I'll go double check

Reply:
this file should now be gone.